pkg/cli/cli.go

```diff
@@ -262,6 +266,9 @@ func NewCmd[T any, PtrT *T](
 	cmd.PersistentFlags().String("log-level", defaultLogLevel, "The log level: debug, info, warn, error ($BATON_LOG_LEVEL)")
 	cmd.PersistentFlags().String("log-format", defaultLogFormat, "The output format for logs: json, console ($BATON_LOG_FORMAT)")
 
+	// Flag for syncing grant expansion
+	cmd.PersistentFlags().BoolP("expand-grants", "e", false, "Enables expanding grants while syncing ($BATON_EXPAND_GRANTS)")
```

Comment:
What is the long term thought about this flag? Do we intend to keep it? Any reason it shouldn't be part of the default behavior?

Reply:
I think the main thing for now is that we can ship it without any behavior being changed. We do some work that we don't need to do unless we are expanding grants. Also we could possibly put on the capabilities object whether or not certain resources are expandable from a top level. We could then skip this step if the connector doesn't support any expandable resources. Main thing is it makes the PR safer in my eyes.